overlord/managers_test.go

```diff
@@ -156,6 +157,11 @@ func verifyLastTasksetIsRerefresh(c *C, tts []*state.TaskSet) {
 func (s *baseMgrsSuite) SetUpTest(c *C) {
 	s.BaseTest.SetUpTest(c)
 
+	// temporary: skip due to timeouts on riscv64
```

Comment:
can we make this a TODO for easier grepping ? otherwise I'm concerned that we will forget about this entirely even when riscv64 hardware catches up and we are actually able to run tests there in a reasonable amount of time

Reply:
I created a jira backlog card as well to remind us to not forget :)